code/game/objects/items/paper/paper.dm

```diff
@@ -50,6 +50,36 @@
 	var/color1 = "#000000"
 	var/color2 = "#FFFFFF"
 
+/obj/item/weapon/paper/entry_permit
+	name = "entry permit"
+	desc = "a permit granting right of entry to a specified country"
+	icon_state = "entry_permit"
+
+/obj/item/weapon/paper/asylum
+	name = "asylum grant"
+	desc = "an official document granting political asylum to the recipient in a specified country"
+	icon_state = "asylum_grant"
+
+/obj/item/weapon/paper/id_supp
+	name = "id supplement"
+	desc = "a small document supplement detailing physical appearance"
+	icon_state = "id_supp"
+
+/obj/item/weapon/paper/vaccine
+	name = "vaccine certificate"
+	desc = "an official medical certificate confirming that a person has been vaccinated against certain disease(s)"
+	icon_state = "vaccine_cert"
+
+/obj/item/weapon/paper/diplomatic_auth
+	name = "diplomatic authorisation"
```

Comment:
`authorization` is the more streamlined and more common spelling, meanwhile, `authorisation` is seen as the alternative spelling.

just to let you know.